### ISSUE TITLE
Dress Policy

### ISSUE THREAD

**Issue by dineshkummarc** [ProcessManagement/ProcessManagement#10]
Dress Policy
>
Monday to Thursday
Casuals (shirt, Pant, Tshirt)
Tshirts
shoes
>
Friday and Saturday
Tshirts
sandals, Flipflop
Slipers

**Comment by denno-secqtinstien** [comment]
Although i oppose such culture, but may be good ! We here have a open culture with no strict rules for dress policy.

**Comment by dineshkummarc** [comment]
referencing issue https://github.com/ProcessManagement/ProcessManagement/issues/8

**Comment by dineshkummarc** [comment]
yes i also oppose such culture but there should be restriction for wearing tear out clothes, old clothes, timeout clothes, dirty clothes.
